Header header
bool warning
bool dock
bool spot
bool dirt_detect
uint8 clean_color
uint8 clean_intensity
